### PR TITLE
[ACR-1113]Add vibration feature

### PR DESCRIPTION
**Pull request by prjung**
Signed-off-by: pr.jung <pr.jung@samsung.com>

### Change Description ###
Add vibration feature

### Bugs Fixed ###
N/A

### API Changes ###
 - ACR: ACR-1113

### CODE REVIEW

File: docs/application/native/guides/device/system.md [hunk 1]
```diff
@@ -197,6 +197,15 @@ The following table lists the download feature keys.
 
 The following table lists the FIDO feature keys.
 
+<a name="vibration"></a>
```
Comment (mijin85cho) [inline docs/application/native/guides/device/system.md]: Is it ok to add vibration key table here? You added `Vibration` as one of `System Keys` in above table.
However this table is added among `Feature Keys` list and even it is not following alphabetical order.

Comment (prjung) [reply docs/application/native/guides/device/system.md]: I made modification.

**Comment by mijin85cho** [comment]
As the modification is simple, English check by Annie is skipped.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**